src/server/enterprise/server/api_server.go

```diff
@@ -310,7 +310,8 @@ func (a *apiServer) Deactivate(ctx context.Context, req *ec.DeactivateRequest) (
 
 	if _, err := col.NewSTM(ctx, a.env.GetEtcdClient(), func(stm col.STM) error {
 		// blind delete
-		return a.enterpriseToken.ReadWrite(stm).Delete(enterpriseTokenKey)
+		a.enterpriseToken.ReadWrite(stm).Delete(enterpriseTokenKey)
```

Comment:
We should probably still catch this error and then ignore it if it's of type `ErrNotFound`, rather than swallowing all errors here.